modules/apps/frontend-view-state/frontend-view-state-service/service.xml

```diff
@@ -51,26 +37,28 @@
 
 		<!-- Other fields -->
 
-		<column name="fvsEntryId" type="long" />
+		<column method-name="FVSEntryId" name="fvsEntryId" type="long" />
+		<column method-name="FDSName" name="fdsName" type="String" />
 		<column name="name" type="String" />
-	</entity>
-	<entity local-service="true" name="FVSEntry" remote-service="false" trash-enabled="false" uuid="true">
-
-		<!-- PK fields -->
-
-		<column container-model="true" name="fvsEntryId" primary="true" type="long" />
-
-		<!-- Audit fields -->
+		<column name="plid" type="long" />
+		<column name="portletId" type="String" />
+		<column name="system" type="boolean" />
```

Comment:
I'm assuming this is to distinguish between user- and admin- created views. Nevertheless, there is a curious situation. Whereas the former ones will be persisted in this service, the latter have separate persistence mechanisms. Either they come from ObjectView (and friends) or are directly mapped onto java classes in the codebase.

So, I don't see how a system view could end up being saved in the `FVSFrontendDataSet` service unless we migrate stuff. Would that be in our future plans?

Reply:
The goal is to have a single persistence for "Custom Views". With this PR, we are setting `FVSFrontendDataSet` to be that persistence. In Objects, when creating a "System View", they may need to call something like:
```java
fvsFrontendDataSetService.addSystemFVSFrontendDataSet(fvsEntryId, fdsName, name);
```

Note that this does not completely invalidate existing Objects persistence. `ObjectViewColumn`, `ObjectViewFilterColumn` and `ObjectViewSortColumn` are all still needed, to define the "Default View" of FDS instance. By "Default View", I mean "Default View" / "View Definition" from our discussion in https://github.com/liferay-frontend/liferay-portal/pull/2526#discussion_r923169418, not the "default" option in Objects UI. It is the same logic as a typical tag FDS, where we will still need `*FDSView.java` and other Java classes. 

But now, with this PR, if Objects team wants to set in UI something in "View State" domain, for example currently selected "contentRenderer" (`table`, `list` or `card`), they will need to add this as a related "System Custom View".